index.html

```diff
@@ -38,6 +39,149 @@ <h2>Introduction</h2>
+          its new location.
+        </p>
+        <p>
+          However, in the context of ActivityPub use cases, objects are almost
+          never encountered by themselves, in isolation. Because ActivityPub
+          is a protocol intended for building distributed social systems, it has
+          two important design aspects: one, authorship and provenance are
+          important, and two, objects are not intended to be created or mutated
+          directly, but are instead wrapped in lifecycle Activities (such as
+          <code>Create</code>, <code>Update</code>, <code>Delete</code>, and
+          others). In distributed system design, this is known as the Event
+          Sourcing pattern, where the state of each object is not stored directly,
+          but is instead constructed from a series of actions taken on the object.
+          The fact that these are social systems brings additional challenges:
+          the series of Actions are always attributed to (performed by) a
+          particular <a>Actor</a>. Also, these Actions typically have an intended
+          audience, which means that for any state change the object undergoes,
+          there may exist any number of stakeholders who must be informed about
+          those changes.
+        </p>
+        <p>
+          Taken together, the design aspects of the ActivityPub protocol present
+          non-trivial challenges, when it comes to data migration and portability.
+          Objects are never encountered by themselves, in isolation. Instead, they
+          exist only as an append-only series of Actions, each performed by an
+          Actor, within a social context (each action typically has intended
+          audiences, as well as other observers and stakeholders).
+          And because of that, moving an object means not only updating its
+          <code>id</code> to a new location, but it also means preserving its
+          chain of actions, authorship, and full social context.
+        </p>
+      </section>
+
+      <section class="informative">
+        <h3>ActivityPub Actor Identity</h3>
+
+        <p>
+          The <a href="https://www.w3.org/TR/activitypub/">ActivityPub 1.0 Specification</a>
+          was intentionally silent on the subject of <a>actor</a> identity,
+          authentication and authorization, leaving those up to <a>server</a>
+          implementers. Multiple implementations arose that modeled <a>actors</a>
+          as traditional user accounts, which controlled various types of data,
+          such as contents (posts, notes, comments, images, likes, etc), related
+          data such as bookmarks, social graphs (lists of followers and following
+          users), actor metadata, moderation decisions (block lists, mutes), and
+          more.
+        </p>
+        <p>
+          - data migration basically always means migration of a given Actor,
+            and all of the data that belongs to it
```

Comment:
```suggestion
            and all of the Activity objects that belong to it
```
(seems a little redundant to say data migration involves moving data haha)

Reply:
It's not only Activity objects though, is it?  It really ought to include block lists and follow lists too -- which is all mentioned in the paragraph above already

Reply:
@lisad exactly, yeah, it's not just Activity objects. (I'll rephrase so that it sounds less redundant, but captures the range)

Reply:
Rephrased, resolving.